e2e_tests/test_browse.py

```diff
@@ -10,25 +10,318 @@ def test_has_title(authenticated_page: Page):
     )
 
 
-def test_sort_dropdown(authenticated_page: Page):
-    authenticated_page.goto("/browse")
-    authenticated_page.select_option("#sort", "body-a")
-    authenticated_page.select_option("#sort", "body-b")
-    authenticated_page.select_option("#sort", "series-a")
-    authenticated_page.select_option("#sort", "series-b")
-    authenticated_page.select_option("#sort", "date-first")
-    authenticated_page.select_option("#sort", "date-last")
+class TestBrowse:
+    def test_browse_sort_dropdown(self, authenticated_page: Page):
```

Comment:
tests are looking good.

I'd just suggest that we could maybe get away without needing these sort dropdown e2e tests. 
I cant see much benefit since the dropdown is tested as part of the sort and filter e2e tests and then all the edge cases of the options are tested in all the standard pytest tests you have written

Reply:
removed sort dropdown test cases as requested